### PR TITLE
Enhancement/issue 24

### PR DESCRIPTION
**Pull request by Urszuja**
Dark theme available, toggle switch working.

### CODE REVIEW

File: frontend/src/components/DarkThemeContext/DarkThemeContext.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export const useThemeUpdate = () => {
+  return useContext(ThemeUpdateContext);
+};
+
+// eslint-disable-next-line react/prop-types
+export const DarkThemeProvider = ({ children }) => {
+  const [darkTheme, setDarkTheme] = useState(false);
+
+  // eslint-disable-next-line react/jsx-no-constructed-context-values
```
Comment (Nefariusek) [inline frontend/src/components/DarkThemeContext/DarkThemeContext.jsx:18]: please disable globally

File: frontend/src/views/App/App.jsx [hunk 2]
```diff
@@ -31,75 +33,12 @@ const paths = [
...
-          },
-        },
-      },
-    },
-  },
-});
-
 const App = () => {
+  const darkMode = useDarkTheme();
+  console.log(darkMode);
```
Comment (Nefariusek) [inline frontend/src/views/App/App.jsx:38]: Please remove console.log

**Review by Nefariusek — CHANGES_REQUESTED** [review]

File: frontend/src/components/DarkThemeContext/DarkThemeContext.jsx (hunk 1 above)
Comment (Urszuja) [reply frontend/src/components/DarkThemeContext/DarkThemeContext.jsx]: done

File: frontend/src/views/App/App.jsx (hunk 2 above)
Comment (Urszuja) [reply frontend/src/views/App/App.jsx]: done

**Pull request merged**